task HTML/arrayTask.js

```diff
@@ -0,0 +1,158 @@
+function isArrayFirst(arr) {
+  return Array.isArray(arr);
+}
+
+function isArraySecond(arr) {
+  return (arr instanceof Array);
+}
+
+function isArrayThird(arr) {
+  return Object.prototype.toString.call(arr) == '[object Array]';
+}
+
+function range(arg1, arg2, arg3) {
```

Comment:
next cases should work:
range(10) [0..9]
range(1, 10) [1..9]
range(1, 10, 3) [1, 4, 7]
range(10, null, 3) [0, 3, 6, 9] *
range(10, null) the same with range(10) *
range(-2, -5) [] *
range(-5, -2) [-5, -4, -3] 
range(-5, -2, 2) [-5, -3]
range(-5, -2, -2) [] *
range(-5, null, 2) [] *
range(-5, null, -2) [0, -2, -4] *
range(-10, -20, -5) [-10, -15] *
range(-20, -10, -5) [] *

    *- have wrong result
    FYI this function can be implemented ~10 lines, firstly you should find step and length of result array (the difficult part of this function)